README.md

```diff
@@ -92,5 +92,14 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for information related to developing the
        * Reach out to the code owner for the necessary credentials.
        * Set up an `.env` file as instructed.
 
+### Docker
+
+1. Install [docker](https://docs.docker.com/v17.09/engine/installation/)
+2. Clone the repository locally, if you haven't already: `git clone https://github.com/mesoscope/cellpack.git`
+3. Ensure that you have valid AWS access key and secret to access the `cellpack-results` S3 bucket, usually stored in a `~/.aws/credentials` file. 
+4. To build the container, run: `docker build -t cellpack_image .` Include `--build-arg local=True` if running locally, omit if running on an EC2 instance connected to EFS volume
```

Comment:
We can remove the second sentence of step 4! This is relevant for octopus, but not needed here :)